Common/Data/Auxiliary/LocalDiskFactorFileProvider.cs

```diff
@@ -71,7 +72,7 @@ public FactorFile Get(Symbol symbol)
                 return GetFactorFile(symbol, symbol.Value, market);
             }
 
-            var mapFile = mapFileResolver.ResolveMapFile(symbol.Value, symbol.ID.Date);
+            var mapFile = mapFileResolver.ResolveMapFile(symbol.Value, DateTime.Today);
```

Comment:
"Maybe what it should be doing is symbol.ID.Symbol and symbol.ID.Date" +1, this sounds right. This way the returned map file wouldn't change depending on the date today.